sites/all/modules/ereol/ereol_inspiration/ereol_inspiration.install

```diff
@@ -32,3 +32,10 @@ function ereol_inspiration_update_7103() {
 function ereol_inspiration_update_7104() {
   module_enable(array('reol_spotlightbox_paragraph'), TRUE);
 }
+
+/**
+ * Enable reol_review_paragraph.
+ */
+function ereol_inspiration_update_7105() {
+  module_enable(array('reol_review_paragraph'), TRUE);
```

Comment:
Dont you need a feature enable/revert also?

Reply:
And you could also add the danish translations through here 
example:
https://github.com/reload/innovationsfonden/blob/develop/sites/all/modules/inno/base/inno_base.install#L532

Reply:
No, features will fix up any differences between site configuration and feature definition, if the feature is not overriden. Feature revert should (yeah, should, reality might be different) only be needed if the feature has been overriden.

Translations has always been an editor problem on this site, no need to change that now.

Reply:
>No, features will fix up any differences between site configuration and feature definition, if the feature is not overriden. Feature revert should (yeah, should, reality might be different) only be needed if the feature has been overriden.
Ok 👍 

>Translations has always been an editor problem on this site, no need to change that now.
Editor problem? Helbredsprofilen had the same problem forever, but now i finally added the translate option to translate in update hooks, and it really makes the client happy that it's ready to go already - but it's your call. It's super simple to set up though ;)